### ISSUE TITLE
Please update BlockLauncher.

### ISSUE THREAD

**Issue by ghost** [zhuowei/MCPELauncher#1517]
@zhuowei please update BlockLauncher.

**Comment by joeyoropesa-dev** [comment]
This project is permanently taken down - and for now there is no known reason for that. Maybe devs are already dead who kept this code up-to-date or etc...

So, better to find some alternative.

**Comment by joeyoropesa-dev** [comment]
... wait - NO WAY - YOU'RE FROM MAGISK??? - anyway, it's good for all of us to close this issue because it's useless.

**Comment by khanhduytran0** [comment]
@admin-elvistrujanovic Nope. He is still on Mastodon/Twitter. It’s just Mojang decided to strip debug symbols from libminecraftpe like an iOS kernel, making it harder than ever to obtain function offsets.

**Comment by joeyoropesa-dev** [comment]
> @admin-elvistrujanovic Nope. He is still on Mastodon/Twitter. It’s just Mojang decided to strip debug symbols from libminecraftpe like an iOS kernel, making it harder than ever to obtain function offsets.

So, is that means that there is still a hope that we will se PojavLauncher updates again?
